flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/metrics/KubernetesOperatorMetricOptions.java

```diff
@@ -23,8 +23,8 @@
 /** Configuration options for metrics. */
 public class KubernetesOperatorMetricOptions {
     public static final ConfigOption<String> SCOPE_NAMING_KUBERNETES_OPERATOR =
-            ConfigOptions.key("metrics.scope.kubernetes-operator")
-                    .defaultValue("<host>.kubernetes-operator.<namespace>.<name>")
+            ConfigOptions.key("metrics.scope.kubernetes_operator")
```

Comment:
nit: the underline format for flink option is not a common style.

Reply:
Thanks, I'll revert this change, it was requested in the PR

Reply:
Reverted